test/README.md

```diff
@@ -0,0 +1,275 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
```

Comment:
(question) we now have SPDX and non-SPDX headers in files, is there a common reconciliation for both? IMHO we should potentially stick to one scheme.

Reply:
I think we should switch to spdx for the entire project, and add a linter to check
for correct spdx headers. The I can post a pr for this quickly to make this change
more consistent with the existing code.

We did this recently in ovirt:
- small project: https://github.com/oVirt/ovirt-imageio/pull/143
- big project: https://github.com/oVirt/vdsm/pull/306

Reply:
I would prefer to stick to the current copyright in the file, as the pattern is common for operators and k8s code in general and hence helps contributors navigate more easily.

spdx allows for better automation and lower frequency for copyright changes, but is it required?

Eitherway we can deal with it in a separate PR, comment was more to understand how to proceed.

Reply:
SPDX added to entire project in #571.